packages/components/src/color-palette/index.native.js

```diff
@@ -51,6 +51,14 @@ function ColorPalette( {
 		'linear-gradient(360deg, rgba(0,0,255,.8) 0%, rgba(0,0,255,0) 70.71%)',
 	];
 
+	const [
+		shouldShowCustomIndicator,
+		setShouldShowCustomIndicator,
+	] = useState(
+		shouldShowCustomIndicatorOption &&
+			( ! isGradientSegment || isCustomGradientColor )
```

Comment:
Do we still need the `isCustomGradientColor` variable? Since  It is the same as  `isGradientColor` I suggest using the `isGradientColor` instead. It is also kinda confusing because you declare the `isCustomGradientColor` a few lines below :)